code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -92,4 +92,21 @@
 		to_chat(H, SPAN_NOTICE("There is nothing there. You feel safe."))
 	return TRUE
 
-
+/datum/ritual/cruciform/base/sense_cruciform
+	name = "Cruciform sense"
+	phrase = "Et si medio umbrae"
+	desc = "very short litany to identify cruciform followers. Targets individuals directly in front of caster or being grabbed by caster."
```

Comment:
Missing capital letter "Very".
"NeoTheology followers" instead of "cruciform followers"
Remove "being", the target needs to be grabbed + grabbing something is instanteneous so there is no really a process of being grabbed.

Reply:
Modified.

Please; feel free to commit these types of changes to my PR's in the future as a maintainer.
I leave the "Allow edits by maintainers" box checked so that you guys can make very easy corrections such as these grammar changes.